Slim/Schema/ResultSet/Contributor.pm

```diff
@@ -23,7 +23,7 @@ sub searchNames {
 	};
 
 	# Bug: 2479 - Don't include roles if the user has them unchecked.
-	if (my $roles = Slim::Schema->artistOnlyRoles('TRACKARTIST')) {
```

Comment:
Oh, what I just suggested is actually doing what you revert here 😂. Wouldn't it be better to always call it this way when needed?